views/index.ejs

```diff
@@ -27,6 +27,31 @@
         }
       }
     }
+    function checkGwOptions(gwOptions) {
+        const outputMessage   = document.querySelector(".output");
+        if(isNotValidGwOptionString(gwOptions)) {
+          outputMessage.innerHTML = `Please check the Gateway Options input : "{${gwOptions}}", it should be of the format `+ `<span class="gwoption">option:value;option2:value2</span>`
+        }
+        else {
+          outputMessage.innerHTML =``;
+        }
+      }
+    function isNotValidGwOptionString(gwOptionsStr) {
+        if (gwOptionsStr){
+          let gwOptions = gwOptionsStr.split(";");
+          let keys=gwOptions.length;   
+          for(each=0;each<keys;each++){              
```

Comment:
By default 'each' will be 'var' which is not a good practice, let's declare it as 'let'